src/parameters/Parameters.jl

```diff
@@ -4,11 +4,45 @@ import ClimaCore as CC
 import Insolation.Parameters as IP
 import Thermodynamics as TD
 import CloudMicrophysics as CM
+import SurfaceFluxes as SF
 
 abstract type AbstractClimaAtmosParameters end
 const ACAP = AbstractClimaAtmosParameters
 
+abstract type AbstractTurbulenceConvectionParameters end
+const ATCP = AbstractTurbulenceConvectionParameters
+
 Base.broadcastable(param_set::ACAP) = tuple(param_set)
+Base.broadcastable(param_set::ATCP) = tuple(param_set)
+
+Base.@kwdef struct TurbulenceConvectionParameters{FT} <: ATCP
+    surface_area::FT
+    max_area::FT
+    min_area::FT
+    tke_ed_coeff::FT
+    tke_diss_coeff::FT
+    tke_surf_scale::FT
+    static_stab_coeff::FT
+    Prandtl_number_scale::FT
+    Prandtl_number_0::FT
+    Ri_crit::FT
+    smin_ub::FT
+    smin_rm::FT
+    min_updraft_top::FT
+    pressure_normalmode_buoy_coeff1::FT
+    pressure_normalmode_drag_coeff::FT
+    entr_tau::FT
+    entr_coeff::FT
+    detr_tau::FT
+    detr_coeff::FT
+    detr_buoy_coeff::FT
+    min_area_limiter_scale::FT
+    min_area_limiter_power::FT
+    max_area_limiter_scale::FT
+    max_area_limiter_power::FT
+    c_smag::FT
+    updraft_number::Int
+end
 
 Base.@kwdef struct ClimaAtmosParameters{FT, TP, RP, IP, MPP, SFP, TCP} <: ACAP
```

Comment:
Should we remove TCP from this struct?

Reply:
I think it should stay there. It's just now it's for the `TurbulenceConvectionParameters`struct I create above. I still make it be part of `ClimaAtmosParameters` the same as the thermo and surface fluxes parameters, etc.